### PR TITLE
test(fs): add `--skip-files`, `--skip-dirs`

### PR DESCRIPTION
**Pull request by wafuwafu13**
## Description

Add tests for `--skip-files` and `--skip-dirs`

## Related issues
- Close https://github.com/aquasecurity/trivy/blob/4ffe74643efca57f1057b979ae419aac2e0c5b56/integration/standalone_tar_test.go#L284

## Related PRs

Remove this section if you don't have related PRs.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [x] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: integration/fs_test.go [hunk 1]
```diff
@@ -40,6 +42,24 @@ func TestFilesystem(t *testing.T) {
...
+			golden: "testdata/gomod-skip-files.json.golden",
+		},
+		{
+			name: "gomod with skip dirs",
+			args: args{
+				securityChecks: "vuln",
+				input:          "testdata/fixtures/fs/gomod",
+				skipDirs:       []string{"/testdata/fixtures/fs/gomod/submod2"},
+			},
+			golden: "testdata/gomod-skip-dirs.json.golden",
```
Comment (knqyf263) [inline integration/fs_test.go:61]: Can we use the same golden file as the test for skip files?

File: integration/standalone_tar_test.go [hunk 2]
```diff
@@ -281,19 +279,6 @@ func TestTar(t *testing.T) {
 				osArgs = append(osArgs, "--input", tt.testArgs.Input)
 			}
 
-			// TODO: test skip files/dirs
-			if len(tt.testArgs.SkipFiles) != 0 {
```
Comment (knqyf263) [inline integration/standalone_tar_test.go:285]: Could you keep them as is? The tests you added are for filesystem scanning and this is for image scanning.

**Review by knqyf263 — COMMENTED** [review]
Thanks for your contribution. I left small comments.

File: integration/fs_test.go (hunk 1 above)
Comment (wafuwafu13) [reply integration/fs_test.go]: fixed.

File: integration/standalone_tar_test.go (hunk 2 above)
Comment (wafuwafu13) [reply integration/standalone_tar_test.go:285]: Add tests https://github.com/aquasecurity/trivy/pull/2984/commits/ce3dcb6d927db4dfc0bd7d9a3b7c526ccbd6eb0e

**Review by knqyf263 — APPROVED** [review]
Great work! Thanks!

**Pull request merged**